gpflow/likelihoods/likelihoods.py

```diff
@@ -158,7 +158,7 @@ def variational_expectations(self, Fmu, Fvar, Y):
 class Gaussian(Likelihood):
     def __init__(self, variance=1.0, **kwargs):
         super().__init__(**kwargs)
-        self.variance = Parameter(variance, transform=positive())
+        self.variance = Parameter(variance, transform=positive(lower=1e-6))
```

Comment:
This line deserves an explanation in documentation.

Reply:
@frgsimpson ^^^

Reply:
@awav I've just added a docstring for the class, let me know if that's what you had in mind

Reply:
Actually, how about just turning this into a keyword argument to the class ?
```python
    def __init__(self, variance=1.0, variance_lower_bound=1e-6, **kwargs):
        super().__init__(**kwargs)
        self.variance = Parameter(variance, transform=positive(lower=variance_lower_bound))
```

Reply:
Done!